scripts/repl.js

```diff
@@ -57,6 +62,86 @@
   };
 
   /*
+   * Utils for working with parse.com for persistence
+   */
+  function ParseUtils () {}
+
+  var __parseLoadingCallbacks = null;
```

Comment:
Why not create a factory function that returns a singleton? Is there even a case where we'd have to `ParseUtils`?